### PR TITLE
[Communication] - Administration - Managed Identity for PhoneNumberClient

### PR DESCRIPTION
**Pull request by minnieliu**

### CODE REVIEW

**Review by jbeauregardb — APPROVED** [review]

File: sdk/communication/Azure.Communication.Administration/README.md [hunk 1]
```diff
@@ -116,6 +116,25 @@ Phone plans come in two types; Geographic and Toll-Free. Geographic phone plans
...
+
+Phone Number Administration clients can be authenticated using connection string acquired from an Azure Communication Resources in the [Azure Portal][azure_portal].
+
+```C# Snippet:CreatePhoneNumberAdministrationClient
+// Get a connection string to our Azure Communication resource.
+var connectionString = "<connection_string>";
+var client = new PhoneNumberAdministrationClient(connectionString);
+```
+
+Phone Number Administration clients also have the option to authenticate using a valid token. With this option,
```
Comment (JoshuaLai) [inline sdk/communication/Azure.Communication.Administration/README.md:129]: nit. This is the terminology we have been using with the other platforms (more or less)
`Phone Number Administration clients also have the option to authenticate with Azure Active Directory Authentication`

**Review by JoshuaLai — APPROVED** [review]

File: sdk/communication/Azure.Communication.Administration/README.md (hunk 1 above)
Comment (minnieliu) [reply sdk/communication/Azure.Communication.Administration/README.md:129]: Thank you, I will update

**Pull request merged**